### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by christopherjturner**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -214,7 +215,47 @@ Any files uploaded by a user will never be sent directly to the frontend service
 
 # Local development
 
-## Setup
+## Developing services that use the CDP-Uploader
+
+If your service is going to use the CDP-Uploader to receive files you may want to start by running the uploader locally. The easiest way to do this is using `docker compose`.
+
+#### Docker Compose
```
Comment (feedmypixel) [inline README.md:222]: Worth adding this in the TOC

File: README.md [hunk 2]
```diff
@@ -7,7 +7,8 @@ Core delivery platform Node.js Frontend Template.
     - [Node.js](#nodejs)
   - [API](#api)
   - [Local development](#local-development)
-    - [Setup](#setup)
+    - [Developing services that use CDP-Uploader](#developing-services-that-use-the-cdp-uploader)
+    - [Setup](#setup-for-developing-cdp-uploader)
     - [Development](#development)
```
Comment (feedmypixel) [inline README.md:12]: A few things missing from the TOC - there are generators online as long as things are not sensitive that you are submitting - https://ecotrust-canada.github.io/markdown-toc/

File: README.md [hunk 3]
```diff
@@ -214,7 +215,47 @@ Any files uploaded by a user will never be sent directly to the frontend service
...
+- cdp-uploader
+
+It will also inject a [start-up script](./compose/start-localstack.sh) into the localstack container that automatically creates the queues and buckets needed by the uploader as well as a test bucket named 'my-bucket'.
+If your service requires its own bucket you can add a line to [start-localstack.sh](./compose/start-localstack.sh) script to create one.
+
+```yaml
+aws --endpoint-url=http://localhost:4566 s3 mb s3://cdp-uploader-quarantine
+aws --endpoint-url=http://localhost:4566 s3 mb s3://my-bucket
+
+## Insert your bucket here, e.g.
```
Comment (feedmypixel) [inline README.md:241]: Is this meant to be a title?

File: README.md [hunk 4]
```diff
@@ -214,7 +215,47 @@ Any files uploaded by a user will never be sent directly to the frontend service
...
+aws --endpoint-url=http://localhost:4566 s3 mb s3://your-service-bucket
+```
+
+Note: the script sets the mock AWS region to be `eu-west-2`. See [aws.env](./compose/aws.env) for the other localstack environment variables.
```
Comment (feedmypixel) [inline README.md:245]: you can use `>` for quotes. Or use one of these:

> [!NOTE]  
> Highlights information that users should take into account, even when skimming.

> [!TIP]
> Optional information to help a user be more successful.

> [!IMPORTANT]  
> Crucial information necessary for users to succeed.

> [!WARNING]  
> Critical content demanding immediate user attention due to potential risks.

> [!CAUTION]
> Negative potential consequences of an action.

File: README.md [hunk 5]
```diff
@@ -214,7 +215,47 @@ Any files uploaded by a user will never be sent directly to the frontend service
...
+If your service is also talking to S3 then it will need to use the same region and credentials when talking to localstack.
```
Comment (feedmypixel) [inline README.md:246]: would an example of this here be handy?

File: README.md [hunk 6]
```diff
@@ -214,7 +215,47 @@ Any files uploaded by a user will never be sent directly to the frontend service
...
+
+If everything has worked as expected the CDP-Uploader will be available on `localhost` port `7337`.
+
+Any other supporting services can be added to the compose file as required.
+
+By default, the CDP-Uploader will be running with its mock scanner enabled. This does not actually virus scan files, rather it simulates a reponse based on filename. If you submit a file with the word `virus` in the name it will be flagged as infected.
+
+#### EICAR files and local development
+
+The test harness does not support `EICAR` virus test files yet, so if you are submitting one and getting a `CLEAN` response back from the uploader's test harness, this is expected. In the real environments EICAR files will work.
```
Comment (feedmypixel) [inline README.md:256]: worth linking to - https://www.eicar.org/download-anti-malware-testfile/ ?

File: README.md (hunk 3 above)
Comment (christopherjturner) [reply README.md:241]: no, its a comment in the example shell file

**Pull request merged**